src/constants/ActionTypes.js

```diff
@@ -16,5 +16,8 @@ export const ACTION_TYPES = {
   HIDE_ADDRESS_WINDOW: 'HIDE_ADDRESS_WINDOW',
   SET_CATEGORY_INVALIDATE: 'SET_CATEGORY_INVALIDATE',
   SET_EVENTS_INVALIDATE: 'SET_EVENTS_INVALIDATE',
-  SET_EVENT_INVALIDATE: 'SET_EVENT_INVALIDATE'
+  SET_EVENT_INVALIDATE: 'SET_EVENT_INVALIDATE',
+  GET_GEOLOCATION: 'GET_GEOLOCATION',
+  SET_GEOLOCATION: 'SET_GEOLOCATION',
+  GET_GEOLOCATION_INVALIDATE: 'GET_GEOLOCATION_INVALIDATE',
```

Comment:
Yes, I noticed that the other epics use SET_{ACTION}_INVALIDATE so I've updated this to match. 

I was thinking it makes sense to be GET_GEOLOCATION_INVALIDATE since it's returned as a result of GET_GEOLOCATION giving an error, but SET_GEOLOCATION_INVALIDATE also makes sense, and I'll keep it consistent with the rest.